src/repository/MongoDb.js

```diff
@@ -45,7 +45,7 @@ class MongoDb extends BaseMongoDb implements IBaseDatabase {
     await this.__runForCollection(
       collectionName,
       async (collection: Object): Promise<*> => {
-        const { page, pageSize = 25, skip, take, ...otherQuery } = query;
+        const { skip = 25, take, ...otherQuery } = query;
```

Comment:
nit: `take` should be 25. Skip should be null

Reply:
ops